socs/agents/hwp_gripper/agent.py

```diff
@@ -659,6 +795,8 @@ def make_parser(parser=None):
                         help='IP of Gripper Beaglebone')
     pgroup.add_argument('--control-port', type=int, default=8041,
                         help='Port for actuator control as set by the Beaglebone code')
+    pgroup.add_argument('--warm-grip-distance', nargs='+', default=[10.0, 10.0, 10.0],
+                        help='Nominal distance for warm grip position')
```

Comment:
If this'll be commonly used in SCFs, can you add it to the example in `docs/agents/hwp_gripper.rst`?

Reply:
This is solved in latest ocs thanks to Jack!